<?xml version="1.0"?>

<launch>

    <arg name="joy_dev" default="/dev/input/js0"/>
    <arg name="config_path"/>

    <node pkg="joy" type="joy_node" name="joy_listener">
        <param name="dev" value="$(arg joy_dev)" />
        <param name="deadzone" value="0.05" /><!-- relative to an axis normalized between -1 and 1 -->
        <param name="autorepeat_rate" value="20" />
    </node>

    <node name="teleop_node" pkg="mobilican_teleop" type="teleop_node.py" output="screen" respawn="false">
        <rosparam file="$(arg config_path)" command="load" />
    </node>


</launch>
